templates/themes/modern/author.html

```
@@ -62,14 +62,14 @@ <h2 class="text-xl font-bold text-on-surface">专栏文章</h2>
     {% if uarts['articles'] %}
         <div class="grid grid-cols-1 sm:grid-cols-2 lg:grid-cols-4 gap-6">
             {% for a in uarts['articles'] %}
-                <div class="group bg-surface dark:bg-surface-variant/20 rounded-2xl overflow-hidden hover:bg-surface-variant/30 flex flex-col transition-all duration-300 hover:shadow-lg hover:-translate-y-1">
+                <div class="group bg-surface dark:bg-surface-variant/20 rounded-3xl overflow-hidden hover:bg-surface-variant/30 flex flex-col transition-all duration-300 hover:shadow-lg hover:-translate-y-1">
                     <!-- Thumbnail -->
                     <a href="/read/cv{{ a['id'] }}"
                        class="relative block aspect-video overflow-hidden">
                         <img loading="lazy"
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)